src/ui/MissingTerm/MissingTermManager.ts

```diff
@@ -5,11 +5,18 @@ import { IDoneBuildingQueryEventArgs } from '../../events/QueryEvents';
 import { IClearBreadcrumbEventArgs, IPopulateBreadcrumbEventArgs } from '../../events/BreadcrumbEvents';
 import { SVGIcons } from '../../utils/SVGIcons';
 import { QueryController } from '../../controllers/QueryController';
+import { IAnalyticsMissingTerm, analyticsActionCauseList } from '../Analytics/AnalyticsActionListMeta';
+import { IAnalyticsClient } from '../Analytics/AnalyticsClient';
 
 export class MissingTermManager {
   static ID = 'MissingTermManager';
   private termForcedToAppear: Array<string>;
-  constructor(root: HTMLElement, private queryStateModel: QueryStateModel, private queryController: QueryController) {
+  constructor(
+    root: HTMLElement,
```

Comment:
Once you have more than one constructor arg, it's better to use an object to pass the args. It makes the args order-independent and easier to expand.